src/v/redpanda/admin_server.h

```diff
@@ -151,7 +179,7 @@ class admin_server {
     template<typename R>
     auto exception_intercepter(
       const ss::sstring& url, const request_auth_result& auth_state) {
-        return [this, url, auth_state](std::exception_ptr eptr) mutable {
+        return [this, url, &auth_state](std::exception_ptr eptr) mutable {
```

Comment:
issue: This needs to move back to the commit where `request_auth_result` has the move operator/constructor created